Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -185,6 +264,55 @@ class StatusBar extends React.Component<{
    */
   static currentHeight = StatusBarManager.HEIGHT;
 
+  /**
+   * Get the current frame (top and height) of the Status Bar.
+   *
+   * Returns a Promise that resolves to a frame object with a `top` and `height`.
+   */
+  static getCurrentFrame(): Promise<{top: number, height: number}> {
+    return StatusBarManager.getCurrentFrame();
+  }
+
+  /**
+   * Adds an event handler. Supported events:
+   *
+   * - `statusBarFrameDidChange`: Fires when the status bar frame has changed.
+   *   The argument to the event handler is a frame object with the keys:
+   *   - `height`: The current height of the status bar.
+   *   - `top`: How far the top of the status bar is offset from the top of the
+   *     screen. This generally only happens when the iOS In-Call Status Bar is
+   *     visible and pushes the app down instead of changing the size.
+   */
+  static addEventListener(
+    eventName: EventName,
+    handler: Function
+  ): {remove: () => void} {
+    invariant(eventName === FRAME_DID_CHANGE_EVENT, `Trying to subscribe to unknown event: "${eventName}"`);
+
+    let listener = StatusBarManagerEventEmitter.addListener(eventName, handler);
+    _subscriptions.set(handler, listener);
+    return {
+      remove: () => StatusBar.removeEventListener(eventName, handler)
+    };
+  }
+
+  /**
+   * Removes the event listener. Do this in `componentWillUnmount` to prevent
+   * memory leaks
+   */
+  static removeEventListener(
```

Comment:
Same here, feels like repeated action. These [3 all](https://github.com/facebook/react-native/blob/1e8f3b11027fe0a7514b4fc97d0798d3c64bc895/Libraries/Components/Keyboard/Keyboard.js#L109-L130) are defined anyway, so it might be worth to just directly call them.

Reply:
Also, I would keep names consistent with `Keyboard` module, that is `addListener` instead of `addEventListener` etc.